backend/typescript/server.ts

```diff
@@ -43,18 +42,57 @@ server.applyMiddleware({
 
 sequelize.authenticate();
 
-// firebaseAdmin.initializeApp({
-//   credential: firebaseAdmin.credential.cert({
-//     projectId: process.env.FIREBASE_PROJECT_ID,
-//     privateKey: process.env.FIREBASE_SVC_ACCOUNT_PRIVATE_KEY?.replace(
-//       /\\n/g,
-//       "\n",
-//     ),
-//     clientEmail: process.env.FIREBASE_SVC_ACCOUNT_CLIENT_EMAIL,
-//   }),
+// const serviceAccountData: Record<string, unknown> = serviceAccount;
+// admin.initializeApp({
+//   credential: admin.credential.cert(serviceAccountData),
+//   databaseURL: "https://uw-blueprint.firebaseio.com",
 // });
 
+admin.initializeApp({
+  credential: admin.credential.cert({
+    projectId: process.env.FIREBASE_PROJECT_ID,
+    privateKey: process.env.FIREBASE_SVC_ACCOUNT_PRIVATE_KEY?.replace(
+      /\\n/g,
+      "\n",
+    ),
+    clientEmail: process.env.FIREBASE_SVC_ACCOUNT_CLIENT_EMAIL,
+  }),
+  databaseURL: "https://uw-blueprint.firebaseio.com",
+});
+const db = admin.database();
+const ref = db.ref("nonProfitApplications");
+app.get("/applications", async (req, res) => {
+  try {
+    const snapshot = await ref.once("value");
+    const applications: any[] = [];
```

Comment:
Can we create a new `Application` type in the `models` folder called `firebase-application.model.ts` and export a correct type instead of casting this as an any?